### PR TITLE
Using get_class instead of ReflectionClass

### PR DESCRIPTION
**Pull request by fntlnz**

### CODE REVIEW

**Comment by fntlnz** [comment]
I initially opened this PR to show that `get_class` could be used instead of `ReflectionClass` to generate the name of the dispatch event.

I did a few benchmarks and I ended up with the fact that `get_class` is not significantly faster than `ReflectionClass`

The `3v4l.org` performance tool gave me the last proof of that

get_class => https://3v4l.org/iLbcY/perf#tabs
ReflectionClass => https://3v4l.org/f8Ufp/perf#tabs

**Comment by gianarb** [comment]
Thanks for your effort and for your analysis!
What do you think about this fix? Maybe the time is similar but get_class is pretty to read?

File: src/Dispatcher.php [hunk 1]
```diff
@@ -85,7 +85,7 @@ private function processRouteInfo(array $dispatch)
         $controller = $this->container->get($dispatch[1][0]);
         $method = $dispatch[1][1];
         $params = $dispatch[2];
-        $function = (new ReflectionClass($controller))->name;
```
Comment (samsonasik) [inline src/Dispatcher.php:88]: remove `use ReflectionClass` from use statements then ;)

Comment (fntlnz) [inline src/Dispatcher.php:88]: I opened the issue just to point out that ReflectionClass is faster than get_class :D

Comment (samsonasik) [inline src/Dispatcher.php:88]: :+1:

**Pull request closed without merging**